lib/functions/RegularFunctions.ts

```diff
@@ -52,36 +53,41 @@ const unaryMinus = {
 const multiplication = {
   arity: 2,
   overloads: declare()
-    .arithmetic((left, right) => left * right)
+  // @ts-ignore
```

Comment:
Are these necessary?

Reply:
I haven't added TS typings yet but I'm going to look into it now!

Reply:
TS gives the error message that .mul, or any other function, is not defined in the module.

Reply:
That's weird. Is it possible that you're using the typings incorrectly? Perhaps try importing like this: `import {mul} from 'decimal.js';`